gqcpy/include/interfaces.hpp

```diff
@@ -89,7 +90,7 @@ void bindBasisTransformableInterface(Class& py_class) {
         .def(
             "transformed",
             [](Type& transformable, const Transformation& T) {
```

Comment:
I noticed that the first argument may become `const`.

```suggestion
            [](const Type& transformable, const Transformation& T) {
```